fury/utils.py

```diff
@@ -176,10 +178,19 @@ def lines_to_vtk_polydata(lines, colors=None):
     vtk_lines.GetData().DeepCopy(numpy_support.numpy_to_vtk(lines_array))
     vtk_lines.SetNumberOfCells(nb_lines)
 
-    is_colormap = False
+    # Create the poly_data
+    poly_data = vtk.vtkPolyData()
+    poly_data.SetPoints(vtk_points)
+    poly_data.SetLines(vtk_lines)
+
     # Get colors_array (reformat to have colors for each points)
     #           - if/else tested and work in normal simple case
-    if colors is None:  # set automatic rgb colors
+    is_colormap = False
+    if colors is False or colors is None:
```

Comment:
`if not colors:` instead. it is shorter

Reply:
I realized this one is necessary, 
since  **colors** can be an array or a list =>
for numpy array it crashes if not using "if colors is False or colors is None:"

For the others "is_colormap", they will be changed since they are only None,False or True